Standards/scs-0402-v1-status-page-openapi-spec-decision.md

```diff
@@ -0,0 +1,107 @@
+If it contains these fields as payload, they SHALL NOT change them.
+
+Requests to updating operations SHOULD contain the minimum of the changed fields, but MAY contain the full object. ID and `Incremental` typed fields MUST follow the same rules as stated above.
+
+### Endpoint naming
+
+The endpoints are named in plural form, even when handeling single objects, to keep uniform paths.
+
+### Phase list
+
+The list of phases an incident can traverse has a crucial order. So it MUST be handled as the given list.
+
+Delete or update operations are FORBIDDEN.
+
+To "change" a phase list, a new one must be created. The old one must be kept. For this mechanic the lists are structured in generations. All references to phases MUST include their generation to ensure correct references.
+
+### Labels
+
+Labes are identifying meta data to components. They do not represent a resource or sub resource of any kind. They are designed as non system critical pieces of information, mainly intented for humans.
+
+They represent a key value pair as structured data for a single component. Same value keys and values do not reference each other necessarily.
+
+### Impact
+
+An impact defines the relation between an incident and a component. A component can be affected by multiple incidents and an incident can affect multiple components. Each of these impacts can have a different type depending on the incident and component, like for example connectivity or performance issues.
+
+To reflect this, each component and incident can have a list of impacts, stating the type of impact and a reference to the incident or component, it refers to.
+
+### Return of `POST` requests
+
+Generally `POST` requests create new resources. These endpoints do not return the new resource, but a unique identifier to the resource e.g. an UUID.
+
+In most cases the new resource won't be used directly after creation. Most often list calls are used. If the new resource is used directly, it can be retrieved by the returned identifier.
+
+Payloads to POST requests SHALL NOT include ID or `Incremental` typed fields, it lies in the jurisdiction of the request handling service to assign IDs and `Incremental`s to objects.
+
+### Return of `PATCH` requestes
+
+Most commonly `PATCH` requests are used to partially or fully change a resource. These requests do not respond with the changed resource, nor an identifier.
+
+Old and new data are existent on the client at that point in time and if they need to be reloaded, the identifier is already known.
+
+### `PATCH` vs `PUT` for updating resources
+
+The `PUT` requests is most commonly used to update full objects, whereas `PATCH` is used for partial updates.
+
+`PATCH` is used as the default method for updating resources because it does not require the full object for an update, but does not discourage the use of the completed object.
```

Comment:
```suggestion
`PATCH` is used as the default method for updating resources because it does not require the full object for an update, but does not discourage the use of the complete object.
```